sirepo/session.py

```diff
@@ -0,0 +1,96 @@
+"""Manage user sessions"""
+
+from pykern.pkdebug import pkdp, pkdlog, pkdexc
+from pykern.pkcollections import PKDict
+import contextlib
+import datetime
+import sirepo.api
+import sirepo.auth
+import sirepo.auth_db
+import sirepo.events
+import sirepo.srcontext
+import sirepo.srtime
+import sirepo.uri_router  #TODO(rorour) remove
+import sirepo.util
+import sqlalchemy
+
+_RENEW_SESSION_TIMEOUT_SECS = 5 * 60
+
+_SRCONTEXT_KEY = __name__
+
+_USER_AGENT_ID_LEN = 8
```

Comment:
This should be the default (32), because it is related to authentication.

Reply:
40fdc6ae4